### PR TITLE
Add `metadata-uid` setup doc about Kubernetes coordinator

### PR DESCRIPTION
**Pull request by wu-sheng**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

This doc should be important for k8s setup if users don't use helm or swck.

### CODE REVIEW

File: docs/en/setup/backend/backend-cluster.md [hunk 1]
```diff
@@ -71,8 +86,29 @@ cluster:
...
 ```
 
+Meanwhile, OAP cluster requires `metadata.uid` as the value of the system environment variable **SKYWALKING_COLLECTOR_UID**
+
+```yaml
+   # Add 
+   containers:
+     # Original configurations of OAP container
+     - name: { { .Values.oap.name } }
+       image: { { .Values.oap.image.repository } }:{{ required "oap.image.tag is required" .Values.oap.image.tag }}
```
Comment (kezhenxu94) [inline docs/en/setup/backend/backend-cluster.md:96]:
```suggestion
     - name: {{ .Values.oap.name }}
       image: {{ .Values.oap.image.repository }}:{{ required "oap.image.tag is required" .Values.oap.image.tag }}
```

**Review by kezhenxu94 — APPROVED** [review]

File: docs/en/setup/backend/backend-cluster.md [hunk 2]
```diff
@@ -71,8 +86,29 @@ cluster:
   # other configurations
 ```
 
+Meanwhile, OAP cluster requires `metadata.uid` as the value of the system environment variable **SKYWALKING_COLLECTOR_UID**
```
Comment (hanahmily) [inline docs/en/setup/backend/backend-cluster.md:89]:
```suggestion
Meanwhile, the OAP cluster requires the pod's UID which is laid at `metadata.uid` as the value of the system environment variable **SKYWALKING_COLLECTOR_UID**
```

**Review by hanahmily — COMMENTED** [review]
some nits

File: docs/en/setup/backend/backend-cluster.md (hunk 2 above)
Comment (wu-sheng) [reply docs/en/setup/backend/backend-cluster.md:89]: Need another PR to fix these words...

**Pull request merged**